mlflow/server/js/src/experiment-tracking/components/App.tsx

```diff
@@ -102,6 +109,10 @@ class App extends Component {
                   </NavLinkV5>
                 </div>
                 <div className='header-links'>
+                  <div style={{ display: 'flex', alignItems: 'center' }} css={{ marginRight }}>
+                    <Switch checked={isDarkTheme} onChange={setIsDarkTheme} />
+                    {isDarkTheme ? <span>Dark theme</span> : <span>Light theme</span>}
+                  </div>
```

Comment:
@jimmyxu-db  please add it to modernized header component as well
https://src.dev.databricks.com/mlflow/mlflow/-/blob/mlflow/server/js/src/common/components/MlflowHeader.tsx?L86